### PR TITLE
Code cleans, fix logic, PBP and custom commands

### PR DESCRIPTION
**Pull request by panshin**
- _read_pid - remove unnecessary eval, add file exist check;
- carp and croak instead of warn and die;
- code style unification;
- fix some logic;

### CODE REVIEW

File: lib/Ubic/Service/Hypnotoad.pm [hunk 1]
```diff
@@ -114,17 +138,17 @@ sub status_impl {
 
 	my $pid = $self->_read_pid;
 
-	if ($self->{'start_time'} and $self->{'start_time'} + $self->{wait_status}{time_to_wait} > time) {
-		return result('broken')		if ! $pid;
+	if ($self->{start_time} and $self->{start_time} - $self->{wait_status}{time_to_wait} > time) {
```
Comment (akarelas) [inline lib/Ubic/Service/Hypnotoad.pm]: Why did you change plus into minus? Is this a typo?

**Comment by panshin** [comment]
Let me explain last commit:

I have removed timeout function from Hypnotoad, because Ubic::Service::Skeleton does the same thing, which will replace Hypnotoad realisation.

When stop_impl invoked and return 'stopping' status, Ubic::Service::Utils#wait_for_status called.

Sorry for my bad english :)

**Comment by panshin** [comment]
Bump

**Comment by akarelas** [comment]
You used 4 spaces where I used tabs. That makes the diff between our two files very large. Can you fix this and use tabs once again? This will make the diff more understandable. I use 1 tab = 4 spaces.

**Comment by panshin** [comment]
Done

**Comment by akarelas** [comment]
What are these changed needed for?

For example, how do carp & croak help you, when compared to warn and die in this case?

**Comment by panshin** [comment]
Carp and croak - Perl Best Practices, only.

This changed fix graceful stop logic - in origin code it works wrong.

Alway get error like this - 
Stopping service.main... 
broken

status_impl - incorrect check the status of service.
